fim_post_processing.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash -e
+
+# Note: This file does you named arguments as one of them is an optional argument (jobLimit)
```

Comment:
Providing an overview of what this script accomplishes here would be aweseome! If not, the naming of the script is pretty self-explanatory. I'd suggest either removing or rewording this line. If rewording, something along the lines of: This file requires one named argument (run name), and one optional argument (job limit).